pkg/namespacestore/validation.go

```diff
@@ -181,6 +186,9 @@ func ValidateNSEmptySecretName(ns nbv1.NamespaceStore) error {
 	switch ns.Spec.Type {
 	case nbv1.NSStoreTypeAWSS3:
 		if len(ns.Spec.AWSS3.Secret.Name) == 0 {
```

Comment:
`Secret` is now optional, so you need to validate it's not nil before dereferencing it

Reply:
sorry. thought this is a pointer. ignore